lexibank_backstromnorthernpakistan.py

```diff
@@ -1,30 +1,108 @@
+import attr
+from clldutils.misc import slug
 from clldutils.path import Path
+from clldutils.text import split_text, strip_brackets
 from pylexibank.dataset import Dataset as BaseDataset
+from pylexibank.dataset import Concept, Language
+from tqdm import tqdm
+import csv
```

Comment:
Make sure to clean/check the imports. Remove unnecessary imports to keep this as clean as possible.